src/main/java/com/volasoftware/tinder/exception/UserDoesNotExistException.java

```diff
@@ -1,5 +1,6 @@
-package com.volasoftware.tinder.exception;public class UserDoesNotExistException extends RuntimeException{
+package com.volasoftware.tinder.exception;
 
+public class UserDoesNotExistException extends RuntimeException{
```

Comment:
How this is different from UserNotFoundException as logic?

Reply:
I thought that I should  make new exception class even thou the cases are identical